cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
```

Comment:
The 'id' field is set to read_only, which is correct since it should not be modified by the user.